packages/webapp/src/containers/Filter/Transactions/index.jsx

```diff
@@ -0,0 +1,86 @@
+/*
```

Comment:
This file is one of the most relevant ones in terms of the actual feature implemented in this PR